### ISSUE TITLE
Fail to replicate the graph task on single GPU for GNN

### ISSUE THREAD

**Issue by maxxxzdn** [smsharma/eqnn-jax#15]
Hey, thanks for putting up the nice dataset!

I, however, didn't manage to replicate the learning curve of a GNN on a single GPU. I ran it as you say:
`python benchmarks/galaxies/train_cosmology.py
`

For me, the model converged to $0.3$ training loss, far from the reported $10^{-3}$. 
`Final checkpoint test loss [[0.21392891 0.39251247]].`

Also, the number of parameters printed was only half the size reported:
`Number of parameters: 700674`

Any ideas why that is the case? Thanks a lot!

**Comment by julballa** [comment]
Thanks for pointing this out!

The target values for $\Omega_m$ and $\sigma_8$ are standardized before training each model, so the final checkpoint losses correspond to the MSE between the normalized values. However, we report the rescaled MSE in the paper. I've added the correct rescaling in #16.

The default hyperparameters have also been updated to match the reported results, so the parameter counts should now also match.
